### DISCUSSION TITLE
github live link

### DISCUSSION THREAD

**Discussion by rakibulhasan0000** [community/community#87314] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

my github account no live link...i create some repositories....but i can't find my pages live link . please help me and oblige me.
i don't see my branch name and others.i cant change it.
![Screenshot_15](https://github.com/community/community/assets/126580936/118d0928-f7a6-4976-a19c-f8b5b09cbbe3)

**Comment by anuragjain-git** [comment, marked as answer]
You have to first set up github pages, 
visit [Github pages documentation](https://docs.github.com/en/pages/quickstart)
Or checkout [Youtube video on github pages](https://www.youtube.com/watch?v=OltY8JIaP-4)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @rakibulhasan0000! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
